core/lib/mcxtx_api.h

```diff
@@ -135,7 +135,15 @@
      * all.  We do not need anything more than word alignment for OP_vldm/OP_vstm,
      * and dr_simd_t has no fields larger than 32 bits, so we have no padding.
      */
+#ifdef AARCH64
+    dr_simd_t simd[MCXT_NUM_SIMD_SVE_SLOTS];
```

Comment:
(We can't change fields in this public structure: but this is the same size as before.  OK.)